src/util/Constants.js

```diff
@@ -1031,6 +1031,24 @@ exports.PrivacyLevels = createEnum([null, 'PUBLIC', 'GUILD_ONLY']);
  */
 exports.PremiumTiers = createEnum(['NONE', 'TIER_1', 'TIER_2', 'TIER_3']);
 
+/**
+ * The entity type of a {@link GuildEvent}:
+ * * SCHEDULED
+ * * ACTIVE
+ * * COMPLETED
+ * * CANCELED
+ * @typedef {string} GuildEventEntityType
```

Comment:
Once the upstream PR lands, please add the `@see` link (maybe add a TODO comment so you don't forget)

Reply:
sure, I'll resolve these convos only when it's done so that I don't forget